pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/JsonToPinotSchema.java

```diff
@@ -78,6 +78,10 @@ public class JsonToPinotSchema extends AbstractBaseAdminCommand implements Comma
       + "JSON string, can be NONE/NON_PRIMITIVE/ALL")
   String _collectionNotUnnestedToJson;
 
+  @CommandLine.Option(names = {"-parseExactBigDecimal"}, description = "Whether to parse exact BigDecimal or parse "
+      + "BigDecimal as doubles, default to false")
+  Boolean _parseExactBigDecimal = false;
```

Comment:
No need to add this option. We should always parse big decimal as is

Reply:
The problem here is that attempting to parse BigDecimal as is may result in doubles (in some occasions) being parsed as BigDecimal and sometimes float being parsed as BigDecimal which is a backward incompatible change.